scss/detail.scss

```
@@ -34,133 +26,347 @@ $container-width: 900px;
   display: flex;
   height: $container-height;
   width: $container-width;
-}
 
-.container-img {
-  position: relative;
-  height: 416px;
-  padding: 24px;
-  width: 309px;
-}
+  .container-img {
+    position: relative;
+    height: 100%;
+    max-width: 309px;
+    padding: 24px;
+    width: 100%;
 
-.container-detail {
-  height: 100%;
-  padding: 24px;
-  width: auto;
-}
+    .badge {
+      display: flex;
+      height: $badge-height;
+      width: $badge-width;
+      position: absolute;
+      transform: rotate(15deg);
+      padding: 10px;
+      right: -5px;
+      top: 0px;
+    }
+  }
 
-.badge {
-  display: flex;
-  height: $badge-height;
-  width: $badge-width;
-  position: absolute;
-  transform: rotate(15deg);
-  padding: 10px;
-  right: -5px;
-  top: 0px;
-}
+  .container-detail {
+    height: 100%;
+    padding: 24px;
+    width: 615px;
 
-.header-detail {
-  align-items: center;
-  border-bottom: 4px solid $green-lemon;
-  display: flex;
-  padding-bottom: 12px;
-  width: 100%;
-}
+    .header-detail {
+      align-items: center;
+      border-bottom: 4px solid $green-lemon;
+      display: flex;
+      padding-bottom: 12px;
+      width: 100%;
 
-.title-book {
-  color: $black-default;
-  font-weight: 700;
-  margin-left: 8px;
-  font-size: 32px;
-  line-height: 32px;
-}
+      .title-book {
+        color: $black-default;
+        font-weight: 700;
+        margin-left: 8px;
+        font-size:  32px;
```

You can remove the added space here
```suggestion
        font-size: 32px;
```